pkg/network/event_windows.go

```diff
@@ -183,7 +183,7 @@ func connDirection(flags C.uint32_t) ConnectionDirection {
 	if (direction & C.FLOW_DIRECTION_OUTBOUND) == C.FLOW_DIRECTION_OUTBOUND {
 		return OUTGOING
 	}
-	return NONE
+	return OUTGOING
```

Comment:
The `if` statement above could be removed

Reply:
I think this way is clearer, that `OUTGOING` is the default value.